openedx/core/djangoapps/user_api/api.py

```diff
@@ -908,6 +911,38 @@ def _add_terms_of_service_field(self, form_desc, required=True):
             },
         )
 
+    def _add_extended_profile_field(self, form_desc, field, required=True):
+        """Add a field to a form description.
+        Arguments:
+            form_desc: A form description
+            field: the field name defined on extended_profile_fields
```

Comment:
Period. But I don't know if it's required, according to Google Style Guide, yes @andrey-canon